scripts/create-component.js

```diff
@@ -82,17 +81,6 @@ try {
     );
   });
 
-  const generatedComponentStoryTemplates = componentStoryTemplates(
```

Comment:
This code is used to generate a new component By `yarn generate <componentName>`, more info here: https://github.com/catho/quantum/blob/master/CONTRIBUTING.MD